scripts/build-i18n-source.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const glob = require('glob');
+const mkdirp = require('mkdirp');
```

Comment:
For consistency, would you mind using the `import` syntax, since we've moved to that elsewhere in the project, and require Node 6 now?